hazelcast/config.py

```diff
@@ -1290,6 +1300,36 @@ def fail_on_indeterminate_operation_state(self, value):
         else:
             raise TypeError("fail_on_indeterminate_operation_state must be a boolean")
 
+    @property
+    def creds_username(self):
+        # type: (_Config) -> str
+        return self._creds_username
+
+    @creds_username.setter
+    def creds_username(self, username):
+        # type: (_Config, str) -> None
```

Comment:
In setters, we generally make some type checks. See the others for example. There are also some unit tests we write to verify valid/invalid inputs for each config option. We should update there too.

Reply:
OK: https://github.com/hazelcast/hazelcast-python-client/pull/446/commits/36946e76b40c9e937d31840499db6bd23f1b14c6